### PR TITLE
Indorse, JET, Qtum, Change Coin

### PR DESCRIPTION
**Pull request by DaveAppleton**
Added tokens IND (indorse) and JET (Jetcoin Institute)

### CODE REVIEW

File: contract-map.json [hunk 1]
```diff
@@ -230,6 +246,23 @@
     "decimals": 18,
     "symbol": "META"
   },
+  "0x9a642d6b3368ddc662CA244bAdf32cDA716005BC": {
+    "name": "Qtum",
+    "logo": "qtum_28.png",
+    "erc20": true,
+    "decimals": 18,
```
Comment (flyswatter) [inline contract-map.json]: According to Etherscan, Qtum has 8 decimals. Can you get a source to corroborate this? https://etherscan.io/token/0x697beac28b09e122c4332d163985e8a73121b97f

File: contract-map.json [hunk 2]
```diff
@@ -230,6 +246,23 @@
...
+    "symbol": "QTUM"
+  },
+  "0x7d4b8Cce0591C9044a22ee543533b72E976E36C3": {
+    "name": "Change Coin",
```
Comment (flyswatter) [inline contract-map.json]: I can't find anywhere they refer to their token as "Change Coin". Might be more accurate to call this "Changebank"?

Comment (david-sg) [reply contract-map.json]: Etherscan shows as ChangeCoin

Comment (GoldenDave) [reply contract-map.json]: https://etherscan.io/address/0x7d4b8cce0591c9044a22ee543533b72e976e36c3#code

Line 218 :
Token name is "Change COIN"

File: contract-map.json (hunk 1 above)
Comment (DaveAppleton) [reply contract-map.json]: That is QRL, Qtum is

https://etherscan.io/token/Qtum#readContract

File: contract-map.json [hunk 3]
```diff
@@ -230,6 +246,23 @@
     "decimals": 18,
     "symbol": "META"
   },
+  "0x9a642d6b3368ddc662CA244bAdf32cDA716005BC": {
+    "name": "Qtum",
+    "logo": "qtum_28.png",
+    "erc20": true,
+    "decimals": 18,
+    "symbol": "QTUM"
```
Comment (flyswatter) [inline contract-map.json]: Good call @DaveAppleton, this value should be `QRL`.

File: contract-map.json (hunk 2 above)
Comment (flyswatter) [reply contract-map.json]: There it is. Thank you.

File: contract-map.json [hunk 4]
```diff
@@ -13,6 +13,22 @@
...
+  "0x8727c112C712c4a03371AC87a74dD6aB104Af768": {
+    "name": "Jetcoin Institute Token",
+    "logo": "JETCOIN28.png",
+    "erc20": true,
+    "symbol": "JET",
+    "decimals": 18
+  },
+  "0xf8e386EDa857484f5a12e4B5DAa9984E06E73705": {
+    "name": "Indorse Token",
+    "logo": "indorse2.png",
```
Comment (flyswatter) [inline contract-map.json]: Logos should be square, sorry this wasn't in the README. Please update the image to be used as a coin logo, ideally graphic only.

Comment (DaveAppleton) [reply contract-map.json]: @david-sg can you pass me a square logo?

File: contract-map.json (hunk 1 above)
Comment (flyswatter) [reply contract-map.json]: Got it, confirmed.

**Comment by DaveAppleton** [comment]
And we are square :-)

**Comment by GoldenDave** [comment]
Am I missing anything?

**Comment by flyswatter** [comment]
Looks good, thanks!

**Pull request merged**